spec/controllers/saml_idp_controller_spec.rb

```diff
@@ -468,6 +468,22 @@ def name_id_version(format_urn)
       end
     end
 
+    context 'service provider has the wrong certs' do
```

Comment:
**question:** just to confirm, this test fails unless you put back the `.first` code?

Reply:
yes, we get the exact same NoMethodError

Reply:
Thanks! I assumed as much but didn't have time to verify locally. Glad you added a test 😌.